gradle/libs.versions.toml

```diff
@@ -5,6 +5,7 @@ hypertrace-grpcutils = "0.5.2"
 hypertrace-framework = "0.1.29"
 lombok = "1.18.20"
 jackson = "2.12.4"
+kafka = "6.0.1"
```

Comment:
can we rename it to confluent as it is confluent platform version and not kafka?

Reply:
Fixed